infrawatch/smart-gateway-operator
# bundle migration

- Migrate latest CSV to manifests directory
- Create initial bundle
- Update Travis-CI to check bundle validation
- No automatic releases for right now
- Add new README.md for release process
- Be verbose when checking bundle validation
- Update build/README.md for validation command
- Smart Gateway Operator v2.0.1 is the base release
